utilities/merge_operators/string_append/stringappend.h

```diff
@@ -24,8 +25,7 @@ class StringAppendOperator : public AssociativeMergeOperator {
   virtual const char* Name() const override;
 
  private:
-  char delim_;         // The delimiter is inserted between elements
-
+  const std::string delim_;  // The delimiter is inserted between elements
```

Comment:
Please remove the "const" here.  Private should be sufficient and it will not work with the Configurable stuff later.